CHANGELOG

```diff
@@ -1,3 +1,19 @@
+lxqt-session-2.1.0 / 2024-11-05
+================================
+ * Reenable locks ettings under wayland.
```

Comment:
Maybe 's/locks ettings/lock settings/'.

Reply:
Thanks for checking these pre-release PRs! Fixed.